### DISCUSSION TITLE
Nachkommastelle Stromzähler wird nicht richtig erkannt

### DISCUSSION THREAD

**Discussion by Tottie78** [jomjol/AI-on-the-edge-device#2962] [Q&A]
Hallo zusammen, 

ich hab meinen alten Stromzähler mit HT und NT ins Home per ESP 32Cam implementiert. funktioniert auch alles super, nur das die Nachkommastelle nicht richtig erkannt wird. Wenn zb das Rädchen zwischen 4 und 5 steht erkennt er xxxxxx,35. Ich hab eine Vermutung das ddies vielleich an der Laufrichtung des Rädchens liegen könnte. 

Nun meine Frage wie kann ich die Laufrichtung ändern ? anbei ein Bild 

Danke 

Thorsten 
![Zähler](https://github.com/jomjol/AI-on-the-edge-device/assets/73481696/5b645b52-4381-4e8e-a372-21da10f9a87d)

**Comment by caco3** [comment]
Die Laufrichtung für die Digits kann nicht eingestellt werden. Du scheinst ein recht exotisches meter zu haben, denn du bist vermutlich der Erste, der mit diesem Problem kommt.

Drehen die anderen Digits auch langsam, oder springen sie? Falls sie springen, lass die Kommastelle einfach weg. Falls sie drehen, wirst Du da die gleichen Probleme kriegen.

@jomjol Hattest Du schon mal so was?

**Reply by Tottie78** [reply]
OK , das dachte ich mir auch schon, das ich dort ein seltenes stück verbaut habe, auch bezüglich der Ziffernschrift fand ich keine gleichen in der Dokumentation. Mit den Kommazahlen hab ich sowieso immer etwas probleme. trotz externer Beleuchtung klappt es nicht. Oder es liegt an den Ziffern. 
Die anderen Digits drehen auch langsam.
